main.tex

```diff
@@ -92,75 +87,69 @@ \section*{Introduction}
+Improvements are needed in our ability to monitor resistance, and gaps must be filled in our knowledge of the molecular mechanisms.
 %%
 
 
 %% Paragraph 2: Introduce VGSC and concept of target-site resistance. What is known about target-site resistance in An. gambiae.
 %
-The voltage-gated sodium channel (VGSC) is the physiological target of pyrethroids and of the organochlorine DDT.
-%
-The VGSC protein is integral to the insect nervous system, involved in the transmission of nerve impulses.
+The voltage-gated sodium channel (VGSC) is the physiological target of pyrethroids and of the organochlorine DDT and is integral to the insect nervous system, involved in the transmission of nerve impulses.
 %
-Both pyrethroids and DDT have a similar mode of action, binding to sites within the protein channel and preventing normal nerve function, causing paralysis (``knock-down'') and then death.
+Both pyrethroids and DDT have a similar mode of action, binding to sites within the protein channel and preventing normal nerve function, causing paralysis (''knock-down'') and then death.
 %
-However, amino acid substitutions at key positions within the channel can alter the interaction between the channel and the insecticide molecule, and thereby substantially increase the dosage of insecticide required for knock-down.
+However, amino acid substitutions at key positions within the channel can alter the interaction with the insecticide molecule, thereby substantially increasing the dosage of insecticide required for knock-down (hence described as knock-down resistance or \emph{kdr}  \cite{Davies2007a}.
 %
-If this tolerance exceeds the dosage present in ITNs or on indoor surfaces following IRS, these interventions may be rendered ineffective.
+If this tolerance exceeds the dosage present in ITNs or on indoor surfaces following IRS, these interventions may be rendered ineffective \cite{N’Guessan2007},\cite{Toe2014}.
 %
-In the African malaria vectors \emph{Anopheles gambiae} and \emph{Anopheles coluzzii}, three substitutions have been found in natural populations and shown to cause pyrethroid and DDT resistance.
+In the African malaria vectors \emph{Anopheles gambiae} and \emph{An. coluzzii}, three substitutions have been found in natural populations and shown experimentally to cause pyrethroid and DDT resistance.
 %
-Two of these substitutions occur in codon 995\footnotemark, with the Leucine $\rightarrow$ Phenylalanine (\texttt{L995F}) substitution prevalent in West and Central Africa, and the Leucine $\rightarrow$ Serine (\texttt{L995S}) substitution found in Central and East Africa.
+Two of these substitutions occur in codon 995, with the Leucine $\rightarrow$ Phenylalanine (\texttt{L995S}) substitution prevalent in West and Central Africa \cite{Martinez-Torres1998},\cite{Silva2014}, and the Leucine $\rightarrow$ Serine (\texttt{L995S}) substitution found in Central and East Africa \cite{Ranson2000},\cite{Silva2014}.
```

Comment:
First L995S should be L995F.

Reply:
What was rationale for dropping the footnote? I thought it might be helpful to put in at the first point in the text where we use a codon number.